### DISCUSSION TITLE
Interesting maps / data visualization

### DISCUSSION THREAD

**Discussion by dabreegster** [alan-turing-institute/uatk-spc#45] [Ideas]
This is just a place to list/track cool data visualizations and maps and similar. Maybe some of them will be inspirations for future projects. Doesn't need to be related to SPC at all; this is just a convenient repo to put the discussion

@mfbenitezp starts us off with https://developers.arcgis.com/javascript/latest/sample-code/visualization-sm-dotdensity/live/index.html?

![image](https://user-images.githubusercontent.com/4097295/215079024-47cacc08-ff11-471c-b125-7c25ffce168b.png)

**Comment by mfbenitezp** [comment]
Daily patterns at the individual level and how a person’s entire day plays out.

https://flowingdata.com/2015/12/15/a-day-in-the-life-of-americans/ 

![image](https://user-images.githubusercontent.com/4097295/215078899-6375b37a-29a9-4e1c-802f-c7cb95fa5b3b.png)
